### PR TITLE
Add link to CFP template doc

### PR DESCRIPTION
**Pull request by lizrice**
Include a link to the CFP template, and instructions to make a copy if you want to use it

Signed-off-by: Liz Rice <liz@isovalent.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_template.md [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
-title: ''
-labels: kind/feature
+title: 'RFE: '
+labels: 'kind/feature'
 assignees: ''
-
 ---
 
-<!--
+## Proposal / RFE
```
Comment (joestringer) [inline .github/ISSUE_TEMPLATE/feature_template.md]: What's an RFE?

File: .github/ISSUE_TEMPLATE/feature_template.md [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
+Include any specific requirements you need
 
-## Proposal / RFE
+**(Optional) Describe your proposed solution**
 
-**Is your feature request related to a problem?**
+Please complete this section if you have ideas / suggestions on how to implement the feature. We strongly recommend discussing your approach with Cilium committers before spending lots of time implementing a change. 
 
-**Describe the solution you'd like**
+For longer proposals, you are welcome to link to an external doc (e.g. a Google doc). We have a [template](https://docs.google.com/document/d/1vtE82JExQHw8_-pX2Uhq5acN1BMPxNlS6cMQUezRTWg/edit) to help you structure your proposal - if you would like to use it, please make a copy and ensure it's publicly visible, and then add the link here.
```
Comment (joestringer) [inline .github/ISSUE_TEMPLATE/feature_template.md]:
```suggestion
For longer proposals, you are welcome to link to an external doc (e.g. a Google doc). We have a [Cilium Feature Proposal template](https://docs.google.com/document/d/1vtE82JExQHw8_-pX2Uhq5acN1BMPxNlS6cMQUezRTWg/edit) to help you structure your proposal - if you would like to use it, please make a copy and ensure it's publicly visible, and then add the link here.
```

**Review by joestringer — APPROVED** [review]
LGTM, thanks!

minor nit that we can include the "Cilium Feature Proposal" (CFP) name when linking the template. Might be worth considering using that nomenclature instead of RFE as well, just to be consistent with existing terms we use.

File: .github/ISSUE_TEMPLATE/feature_template.md [hunk 3]
```diff
@@ -1,24 +1,25 @@
 ---
 name: Feature Request
 about: Suggest a feature to Cilium
-title: ''
-labels: kind/feature
+title: 'RFE: '
```
Comment (lizrice) [inline .github/ISSUE_TEMPLATE/feature_template.md]:
```suggestion
title: 'CFP: '
```

File: .github/ISSUE_TEMPLATE/feature_template.md (hunk 1 above)
Comment (lizrice) [inline .github/ISSUE_TEMPLATE/feature_template.md]:
```suggestion
## Cilium Feature Proposal
```

Comment (lizrice) [reply .github/ISSUE_TEMPLATE/feature_template.md]: Ha, yes OK, let's just use CFP

**Pull request merged**